build/ci/set-up-mms.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+# Copyright 2021 MongoDB Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+while getopts 'h:' opt; do
+  case ${opt} in
+  h) hostsFile="${OPTARG}" ;; # Output of Evergreen host.list
+  *) exit 1 ;;
+  esac
+done
+
+hosts=$(
+  cat <<EOF | python - "${hostsFile}"
+import sys
+import json
+with open(sys.argv[1]) as hostsfile:
+    hosts = json.load(hostsfile)
+    for host in hosts:
+        print(host["dns_name"])
+EOF
+
+)
+
+cd ..
+cd ..
+
+for host in ${hosts}; do
+
+echo "set service"
+  ./bin/mongocli config set service ops-manager
+
+echo "set ops_manager_url"
+  ./bin/mongocli config set ops_manager_url "http://${host}:9080/"
+done
+
+echo "create first user"
+./bin/mongocli om owner create --firstName evergreen --lastName evergreen --email evergreenTest@gmail.com --password "evergreen1234_" -o json > apikeys.json
+
+cat apikeys.json
+publicKey=$(
+  cat <<EOF | python - apikeys.json
+import sys
+import json
+with open(sys.argv[1]) as jsonfile:
+    user = json.load(jsonfile)
+    print(user["programmaticApiKey"]["publicKey"])
+EOF
+
+)
+
+privateKey=$(
+  cat <<EOF | python - apikeys.json
+import sys
+import json
+with open(sys.argv[1]) as jsonfile:
+    user = json.load(jsonfile)
+    print(user["programmaticApiKey"]["privateKey"])
+EOF
+
+)
+
+echo "set public_api_key"
+./bin/mongocli config set public_api_key "${publicKey}"
+
+echo "set private_api_key"
+./bin/mongocli config set private_api_key "${privateKey}"
+
+echo "create organization"
+./bin/mongocli iam organizations create myOrg -o json > organization.json
+
+cat organization.json
+organizationID=$(
+  cat <<EOF | python - organization.json
+import sys
+import json
+with open(sys.argv[1]) as jsonfile:
+    org = json.load(jsonfile)
+    print(org["id"])
+EOF
+
+)
+
+# This mongocli command returns an error when the user has been created with mongocli om owner create but the project is created anyway
+# More info: https://jira.mongodb.org/browse/CLOUDP-76824
+set +e
+echo "create project"
+./bin/mongocli iam projects create myProj --orgId "${organizationID}" -o json
+
+set -e
+echo "get project id"
+./bin/mongocli iam project list --orgId "${organizationID}" -o json > project.json
+cat project.json
```

Comment:
I have to call the `project list` because the previous command fails with 500 error https://jira.mongodb.org/browse/CLOUDP-76824

Reply:
a workaround for that error should be to invite the global owner to the first org

Reply:
I have tried here https://evergreen.mongodb.com/task_log_raw/mongocli_master_e2e_generic_ops_manager_generic_e2e_patch_60349802f2b9d4a011e9f9f8f189cd2a814b0911_6012ff01850e6109c8a31a00_21_01_28_18_14_36/9?type=T but I got the same error.

```
 [2021/01/29 09:57:34.196] invite owner to organization
 [2021/01/29 09:57:34.196] The user 'evergreentest1@gmail.com' has been invited.
 [2021/01/29 09:57:34.196] Invited users do not have access to the project until they accept the invitation.
 [2021/01/29 09:57:34.391] Error: POST http://ec2-54-226-39-245.compute-1.amazonaws.com:9080/api/public/v1.0/groups: 500 (request "UNEXPECTED_ERROR") Unexpected error.
 [2021/01/29 09:57:34.516] create project
 [2021/01/29 09:57:34.516] get project id
```

Reply:
you can set up om to skip invitations, https://docs.opsmanager.mongodb.com/current/reference/configuration/index.html#mms.user.bypassInviteForExistingUsers

Also please don't use emails you don't control `evergreentest1@gmail.com`

Reply:
Could you suggest an email that I can use here?